doc/source/user_guide/groupby.rst

```diff
@@ -1350,35 +1339,53 @@ The returned dtype of the grouped will *always* include *all* of the categories
 
    s = (
        pd.Series([1, 1, 1])
-       .groupby(pd.Categorical(["a", "a", "a"], categories=["a", "b"]), observed=False)
+       .groupby(pd.Categorical(["a", "a", "a"], categories=["a", "b"]), observed=True)
```

Comment:
This demonstrates unobserved categories are being kept, so I think it makes more sense to show with `observed=True`